src/main/java/com/example/hbs/service/HotelService.java

```diff
@@ -98,6 +103,11 @@ public HotelResponseDto deleteHotel(Long id, Long userId) {
         Hotel hotel = hotelRepository.findById(id).orElse(null);
         Optional.ofNullable(hotel).orElseThrow(() -> new HbsException("Hotel not found"));
         checkIfUserIsOwner(hotel, userId);
+        Pageable page = PageRequest.of(0, 10);
+        List<Booking> booking = bookingRepository.findAllBookingofHotel(id, page).getContent();
+        if(!booking.isEmpty()){
+            throw new HbsException("Hotel with bookings greater than 0 can not be deleted");
+        }
```

Comment:
hotel.getBookings should work

Reply:
Okay

Reply:
I think this is same, just my function name is different